### DISCUSSION TITLE
Design the razor based template on admin panel of orchard core like liquid

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10602] [Q&A]
How i design the razor based template in admin panel of orchard core like liquid:

![liquid-template](https://user-images.githubusercontent.com/5202291/139580205-da47082b-b21a-4d0c-873b-04a1de0dd45b.PNG)

**Comment by ns8482e** [comment]
You define razor view in your project. Admin template feature only supports liquid

**Reply by saleemhaq** [reply]
> You define razor view in your project. Admin template feature only supports liquid
![widgetIssue](https://user-images.githubusercontent.com/5202291/139586840-94171644-68ed-4560-9d89-e882b6020fb1.png)

**Reply by ns8482e** [reply]
Not sure if I quite understand your problem - 

However, to elaborate - 
1. The `Widgets` you add in layers are added to different zone that you define in site settings - 
2. These zones to be rendered from layout of the theme and if the theme is active then it will be displayed in your front end.

To render these zones in  theme's layout - use RenderSection in layout.cshtml https://docs.orchardcore.net/en/latest/docs/reference/modules/Layers/#zones.

**Reply by saleemhaq** [reply]
> Not sure if I quite understand your problem -
> 
> However, to elaborate -
> 
> 1. The `Widgets` you add in layers are added to different zone that you define in site settings -
> 2. These zones to be rendered from layout of the theme and if the theme is active then it will be displayed in your front end.
> 
> To render these zones in theme's layout - use RenderSection in layout.cshtml https://docs.orchardcore.net/en/latest/docs/reference/modules/Layers/#zones.

I define zone name as content I make widget and just call it widget render properly but also this text come under <h1> I dont know why?
